.github/actions/get-configuration/action.yaml

```diff
@@ -0,0 +1,38 @@
+name: Get configuration
+description: Defines configuration variables such as versions. Exposes globally needed environment variables.
+outputs:
+  k8s_version:
+    description: The version of k8s to use. For example, 1.28.7.
+    value: ${{ steps.define-variables.outputs.k8s_version }}
+  istio_version:
+    description: The version of Istio to install. For example, 1.20.3.
+    value: ${{ steps.define-variables.outputs.istio_version }}
+  k3d_version:
+    description: The version of k3d to install. For example, 5.6.0.
+    value: ${{ steps.define-variables.outputs.k3d_version }}
+  cert_manager_version:
+    description: The version of cert-manager to deploy. For example, 1.13.3.
+    value: ${{ steps.define-variables.outputs.cert_manager_version }}
+  klm_version_tag:
+    description: The version tag for the KLM image. For example, PR-123.
+    value: ${{ steps.define-variables.outputs.klm_version_tag }}
+  klm_image_repo:
+    description: The repository for the KLM image. For example, dev.
+    value: ${{ steps.define-variables.outputs.klm_image_repo }}
+runs:
+  using: "composite"
+  steps:
+    - name: Define variables
+      id: define-variables
+      shell: bash
+      run: |
+        echo "k8s_version=${{ github.event.inputs.k8s_version || '1.28.7' }}" >> $GITHUB_OUTPUT
+        echo "istio_version=1.20.3" >> $GITHUB_OUTPUT
+        echo "k3d_version=5.6.0" >> $GITHUB_OUTPUT
+        echo "cert_manager_version=1.13.3" >> $GITHUB_OUTPUT
+        echo "klm_version_tag=PR-${{ github.event.pull_request.number }}" >> $GITHUB_OUTPUT
+        echo "klm_image_repo=dev" >> $GITHUB_OUTPUT
```

Comment:
comment: these here used to be a conditional based on whether the job is running for a `main` build or a `PR` build. Since we removed the post `main` build, I removed this conditional now.